tests-aftership/README.md

```diff
@@ -0,0 +1,45 @@
+# Tests for scrapyrt-aftership
+
+All tests for aftership-courier-api are in folder tests-aftership
+
+There are 3 different test files: test\_aftership\_resource.py, test\_AfterShipErrorPage.py, test\_HTTP\_status\_code\_and\_meta\_code.py, each of it tests different part of scrapyrt for AfterShip.
+
+## Install the dependency
+
+First you may need to install the py.test module, it will make the result more beautiful and readable.
```

Comment:
You don't need to do this.  To install development dependencies, you do `pipenv install -d` at the root directory of the project.